### ISSUE TITLE
Issue when application crash and return to mainactivity Pawns starts another notification

### ISSUE THREAD

**Issue by Ayman-apps** [pawns-app/android-pawns-sdk-demo#6]
To reproduce the issue.
- Use Pawns Foreground service with custom notification
- Application get exception in another activity (not main)
- Application returns to mainactivity  (as if restarting the app class)
-  A new Pawns notification appears with the default data defined in pawns

**Comment by laurynas-pawns** [comment]
Could you tell me in which class you are initialising our SDK and building Pawns.Builder?

**Comment by Ayman-apps** [comment]
Both are done in the application class

**Comment by laurynas-pawns** [comment]
Is it done on **onCreate** method of application class?

**Comment by Ayman-apps** [comment]
Yes for both initialization and startsharing

**Comment by laurynas-pawns** [comment]
TLDR; Fix will be launched in the following version 1.4.0 

When first doing this functionality I did not take into an account that service can get recreated by OS. This means, that `startSharing` method is not suitable for this kind of task, because it is designed to be launched later on in the lifecycle. `Notification `and `NotificationId` values will have to be passed to `Pawns.Builder` in `Application` class while initialising SDK.

**Comment by Ayman-apps** [comment]
Great, when is it expected to release 1.4.0?
